stdpopsim/catalog/arabidopsis_thaliana.py

```diff
@@ -80,21 +80,15 @@
 ###########################################################
 
 
-class ArabidopsisThalianaModel(stdpopsim.Model):
-    """
-    TODO: documentation
-    """
-    species = _species
-
-
 # FIXME this documentation needs to be filled out.
-class _Durvasula2017MSMC(ArabidopsisThalianaModel):
+class _Durvasula2017MSMC(stdpopsim.Model):
     id = "SMA_1pop"
     name = "South Middle Atlas population size history"
     description = """
         Model estimated from two homozygous individuals from the South Middle Atlas
         using MSMC (TODO: more detail).
     """
+    species = _species
```

Comment:
Are we actually using the species attribute in client code? If not, I think we should drop it. We can use the ``_species`` variable to get access to the generation times, etc.